src/components/BuySellForm/BuySellAmount/BuySellAmount.scss

```diff
@@ -1,4 +1,19 @@
 .buy-sell-amount {
```

Comment:
Now the fields and adding payment method card / selecting a payment method are disabled when you're balance is low. I also fixed the bug where if your available balance is low, you couldn't place an order from Buy Modal, but in prod, you can

Uploading Screen Recording 2024-08-14 at 11.12.13 AM.mov…

![Screenshot 2024-08-14 at 11 19 17 AM](https://github.com/user-attachments/assets/c262209a-f4d1-4c7a-b1f4-6b12fe627a20)